src/types/message.ts

```diff
@@ -103,6 +80,30 @@ export interface StopMessage extends MessageBase {
   type: 'stop';
 }
 
+export interface GeofenceMessage extends MessageBase {
+  type: 'geofence';
+
+  /**
+    * Geofencing coordinates in the form of a polygon.
+    */
+  geofence: {
+    keepOut: [number, number][], // Both keepOut and KeepIn arrays are filled with
```

Comment:
lets put a `/** */` type of comment above keepIn and keepOut so that when you hover over the variables, then the description shows